src/libs/ReportUtils.ts

```diff
@@ -2941,6 +3000,68 @@ function getReportActionMessage(reportAction: ReportAction | EmptyObject, parent
     return reportAction?.message?.[0]?.text ?? '';
 }
 
+/**
+ * Get the title for an invoice room.
+ */
+function getInvoicesChatName(report: OnyxEntry<Report>): string {
+    const invoiceReceiver = report?.invoiceReceiver;
+    const isIndividual = invoiceReceiver?.type === CONST.INVOICE_RECEIVER_TYPE.INDIVIDUAL;
+    const invoiceReceiverAccountID = isIndividual ? invoiceReceiver.accountID : -1;
+    const policyID = isIndividual ? '' : invoiceReceiver?.policyID ?? '';
+    let isReceiver = false;
+
+    if (isIndividual && invoiceReceiverAccountID === currentUserAccountID) {
+        isReceiver = true;
+    }
+
+    if (!isIndividual && PolicyUtils.isPolicyMember(policyID, allPolicies)) {
+        isReceiver = true;
+    }
+
+    if (isReceiver) {
+        return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]);
+    }
+
+    if (isIndividual) {
+        return PersonalDetailsUtils.getDisplayNameOrDefault(allPersonalDetails?.[invoiceReceiverAccountID]);
+    }
+
+    return getPolicyName(report, false, allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`]);
+}
+
+/**
+ * Get the subtitle for an invoice room.
+ */
+function getInvoicesChatSubtitle(report: OnyxEntry<Report>): string {
```

Comment:
Please, double check this function, considering my comments to `getInvoicesChatName` above

Reply:
Done - 23dd180.